webapp/ng_tests/controllers/register_controller_test.js

```diff
@@ -1,4 +1,4 @@
-'use strict'
+'use strict';
```

Comment:
didn't know this was needed. I would check all other files just in case it was missed everywhere